workloads/controllers/virtual-machine-replica-set.md

```diff
@@ -1,66 +1,66 @@
+The _VirtualMachineInstanceReplicaSet_ allows us to specify a _VirtualMachineInstanceTemplate_ in `spec.template`. It consists of `ObjectMetadata` in `spec.template.metadata`, and a `VirtualMachineInstanceSpec` in `spec.template.spec`. The specification of the virtual machine is equal to the specification of the virtual machine in the `VirtualMachineInstance` workload.
 
 `spec.replicas` can be used to specify how many replicas are wanted. If unspecified, the default value is 1. This value can be updated anytime. The controller will react to the changes.
 
-`spec.selector` is used by the controller to keep track of managed virtual machines. The selector specified there must be able to match the virtual machine labels as specified in `spec.template.metadata.labels`. If the selector does not match these labels, or they are empty, the controller will simply do nothing except from logging an error. The user is responsible for not creating other virtual machines or _VirtualMachineReplicaSets_ which conflict with the selector and the template labels.
+`spec.selector` is used by the controller to keep track of managed virtual machines. The selector specified there must be able to match the virtual machine labels as specified in `spec.template.metadata.labels`. If the selector does not match these labels, or they are empty, the controller will simply do nothing except from logging an error. The user is responsible for not creating other virtual machines or _VirtualMachineInstanceReplicaSets_ which conflict with the selector and the template labels.
 
-## Exposing a VirtualMachineReplicaSet as a Service
+## Exposing a VirtualMachineInstanceReplicaSet as a Service
 
-A VirtualMachineReplicaSet could be exposed as a service. When this is done, one of the VirtualMachines replicas will be picked for the actual delivery of the service.
+A VirtualMachineInstanceReplicaSet could be exposed as a service. When this is done, one of the VirtualMachineInstances replicas will be picked for the actual delivery of the service.
 
-For example, exposing SSH port (22) as a ClusterIP service using virtctl on a VirtualMachineReplicaSet:
+For example, exposing SSH port (22) as a ClusterIP service using virtctl on a VirtualMachineInstanceReplicaSet:
 
 ```bash
-$ virtctl expose vmrs vm-ephemeral --name vmservice --port 27017 --target-port 22
+$ virtctl expose vmirs vmi-ephemeral --name vmiservice --port 27017 --target-port 22
 ```
 
-All service exposure options that apply to a VirtualMachine apply to an VirtualMachineReplicaSet. See [Exposing VirtualMachine](http://www.kubevirt.io/user-guide/#/workloads/virtual-machines/expose-service) for more details.
+All service exposure options that apply to a VirtualMachineInstance apply to an VirtualMachineInstanceReplicaSet. See [Exposing VirtualMachineInstance](http://www.kubevirt.io/user-guide/#/workloads/virtual-machines/expose-service) for more details.
```

Comment:
one more filter to run. 's/[an Virtual/a Virtual' 

There are a few instances of this left